help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -0,0 +1,192 @@
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+* When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you will need to manually map Platform segments to the custom attribute in Salesforce.
+* To do this, select each segment, then enter the corresponding custom attribute from Salesforce in the **[!UICONTROL Mapping ID]** field.
+
+    > [!IMPORTANT]
+    >
+    > * The value used for the Mapping ID should exactly match the name of the custom attribute created within Salesforce under the "Email Demographics" attribute-set.
+
+* An example is shown below:
+![Schedule segment export](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Browse Destinations](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Destinations Dataflow Run](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Destinations Activation Data](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Segment](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/segment.png)
+
+1. Login to the Salesforce Marketing Cloud website. Then navigate to the **[!DNL Audience Builder]** > **[!DNL Contact Builder]** > **[!DNL All contacts]** > **[!DNL Email]** page and check if the profiles from the segment have been added.
+![Salesforce Contacts](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/contacts.png)
+
+1. To check if any profiles have been updated, navigate to the **[!DNL Email]** page check if the attribute values for the profile from the segment have been updated. 
+![Salesforce Contacts](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/contact-detail.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
```

Comment:
Same comment about this section as here: 
https://github.com/AdobeDocs/experience-platform.en/pull/60/files#r921531186